examples/language-modeling/README.md

```diff
@@ -195,7 +197,9 @@ python run_mlm.py \
     --do_eval \
     --output_dir /tmp/test-mlm \
     --use_habana \
-    --use_lazy_mode \
+    --use_lazy_mode False \
+    --torch_compile \
+    --torch_compile_backend hpu_backend \
     --use_hpu_graphs_for_inference \
     --gaudi_config_name Habana/roberta-base \
```

Comment:
Roberta Base for now is supported in lazy mode, Roberta Large to be supported in torch.compile

Reply:
Fixed in new commit "Changes after review". Rebased to latest habana-main.